D7-to-D8.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+read -p "Enter the D7 site shortname on ACSF (e.g., \"opinion\"; no quotes): " acsfd7shortname
+read -p "Enter the D8 site shortname on ACSF (e.g., \"opinion\"; no quotes): " acsfd8shortname
+read -p "Enter the stanford.edu hostname assigned to the D7 site (e.g., \"opinion\"; no quotes): " hostname
+read -p "Enter the ACSF site id of the D7 site (e.g., \"1234\"; no quotes): " acsfd7siteid
+read -p "Enter the ACSF site id of the D8 site (e.g., \"5678\"; no quotes): " acsfd8siteid
+netdb node name --add $acsfd8shortname.cardinalsites.acsitefactory.com --remove $acsfd7shortname.cardinalsites.acsitefactory.com $acsfd7shortname.cardinalsites.acsitefactory.com
```

Comment:
What happens if the site has an alias too?

Reply:
The alias stays with it. Kinda handy.

Reply:
There's no `netdb node rename` command as far as I could see. Tested and this works.

Reply:
Huh. Thats cool then